samples/addons/grafana.yaml

```diff
@@ -19,10 +19,10 @@ metadata:
   name: grafana
   namespace: istio-system
   labels:
-    helm.sh/chart: grafana-5.0.7
+    helm.sh/chart: grafana-5.1.2
     app.kubernetes.io/name: grafana
     app.kubernetes.io/instance: grafana
-    app.kubernetes.io/version: "6.6.2"
+    app.kubernetes.io/version: "7.0.1"
```

Comment:
I think we need to be on 7.0.2 -- there was a security patch release today.

Reply:
Good point. I updated this main as a result of that and then didn't actually use that version... I just updated it